src/vega.schema.ts

```diff
@@ -751,16 +779,16 @@ export interface VgMarkConfig {
   fontSize?: number;
 
   /**
-   * The font style (e.g., italic).
+   * The font style (e.g., `italic`).
    */
   fontStyle?: FontStyle;
   /**
-   * The font weight (e.g., `"normal"`, `"bold"`, `900`).
+   * The font weight (e.g., `bold`).
    */
   fontWeight?: FontWeight | FontWeightNumber;
 
   /**
-   * Placeholder Text
+   * Placeholder text if the `text` channel is not specified (`"Abc"` by default).
```

Comment:
it didn't mention the default for `text` in `defaultConfig`, so I just delete (`"Abc"` by default).